olm/build-bundles.sh

```diff
@@ -1,40 +1,77 @@
 #!/usr/bin/env bash
-# Usage:
-#   ./olm/build-bundles.sh -r <release as x.y.z> -b <branch-name>
+#
+# Description
+#
+# This scripts generates the OLM manifests required to install an operator from a custom catalog.
+# These are not required to install an operator from the OperatorHub.
+#
+# The images are published under docker.stackable.tech/sandbox since they are only needed during development.
+#
+# This script makes the following *assumptions*:
+#
+# - There is a clone of the opernshift-certified-operators repository in the folder passed as -c argument.
+#   This is the same as the build-manifests.sh script.
+#
+# - The operator manifests for the given version have been generated with the build-manifests.sh script
+#   and are available in that repository under operators/<operator>/version/manifests.
+#
+# - If a deployment is also done (with -d) then the namespace called "stackable-operators" is available.
+#
+# Usage
+#
+#   ./olm/build-bundles.sh [options]
+#
+# Options
+#
+#   -c <location of the RH cert operators repo>
 #   -r <release>: the release number (mandatory). This must be a semver-compatible value to patch-level e.g. 23.1.0.
-#   -b <branch>: the branch name (mandatory) in the (stackable forked) openshift-certified-operators repository.
-#   -o <operator-name>: the operator name (mandatory) e.g. airflow-operator.
-#   -d <deploy>: optional flag for catalog deployment.
+#   -o <operator-name>: the operator name (mandatory) e.g. airflow. Without the "-operator" suffix!
+#   -d <deploy>: optional flag for operator deployment.
+#
+# Example
+#
+#   ./olm/build-bundles.sh \
+#       -r 23.4.1 \
+#       -o secret \
+#       -c $HOME/repo/stackable/openshift-certified-operators \
+#       -d
 #
-# e.g. ./olm/build-bundles.sh -r 23.4.1 -b secret-23.4.1 -o secret-operator -d
 
 set -euo pipefail
 set -x
 
-
 SCRIPT_NAME=$(basename $0)
 
 parse_inputs() {
-  INITIAL_DIR="$PWD"
-
-  VERSION=""
-  BRANCH=""
-  OPERATOR_NAME=""
-  DEPLOY=false
-
-  while [[ "$#" -gt 0 ]]; do
-      case $1 in
-          -r|--release) VERSION="$2"; shift ;;
-          -b|--branch) BRANCH="$2"; shift ;;
-          -o|--operator) OPERATOR_NAME="$2"; shift ;;
-          -d|--deploy) DEPLOY=true ;;
-          *) echo "Unknown parameter passed: $1"; exit 1 ;;
-      esac
-      shift
-  done
-
-  # e.g. "airflow" , "spark-k8s"
-  OPERATOR=$(basename "${OPERATOR_NAME}" | rev | cut -d- -f2- | rev)
+	INITIAL_DIR="$PWD"
+
+	VERSION=""
+	BRANCH=""
```

Comment:
is this ignored completely? If so we can remove it.

Reply:
Done: https://github.com/stackabletech/stackable-utils/pull/61/commits/2328728e8fe57cad2da156dd1a6429414f1546bc